packages/dashboards/tests/acceptance/dashboards-test.js

```diff
@@ -383,6 +383,12 @@ module('Acceptance | Dashboards', function(hooks) {
     await click('.add-widget .btn');
     await click('.add-widget-modal .add-to-dashboard');
 
+    assert.deepEqual(
+      findAll('.navi-report-widget__footer .btn').map(e => e.textContent.trim()),
+      ['Run'],
+      'Before running a new widget, only the "Run" button is shown ("Save" and "Revert" are hidden)'
+    );
+
     // Fill out request
     await click($('.checkbox-selector--metric .grouped-list__item:contains(Total Clicks) .grouped-list__add-icon')[0]);
     await click('.navi-report-widget__run-btn');
```

Comment:
Copy
```js
    assert.deepEqual(
      findAll('.navi-report-widget__footer .btn').map(e => e.textContent.trim()),
      ['Run'],
      'Before running a new widget, only the "Run" button is shown ("Save" and "Revert" are hidden)'
    );
```
but check for expected buttons after running

Reply:
only the "save" should be added ("revert" is for already saved widgets) and we're clicking it right below